src/changes-backwards-incompatible.apib

```diff
@@ -3,6 +3,10 @@
 
 We list all backwards-incompatible changes here. As described above, new additions and forwards-compatible changes don’t need a new API version and can be found [here](#additions).
 
+#### 2019-12-11
+
+- The task assignee is now nullable.
```

Comment:
Q: How will we maintain compatibility for consumers on earlier versions? If a task is created without assignee, a consumer on an earlier version calling `tasks.info` will get `null` as assignee, which is a BC break, right?

Reply:
After discussing this with @stivni we agreed to return the task assignee as `{ type: user, id: null }` for older versions, in case the task is unassigned. It's still not perfect, since previously you would always get a non-null id, but at least it respects the format. 

The reality is that, in this case, we didn't find a way to not break compatibility and allow unassigned tasks at the same time. We can only mitigate this by keeping the returned format for older versions.

However, I'm not sure how to document this here.